src/tools/analyzer.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+permalink: /tools/analyzer
+title: "Static Analyzer"
+description: "The tool that analyzes and validates your Dart code."
+---
+
+The static analyzer evaluates your Dart code,
+checking for errors and warnings that are specified in the
+[Dart Language Specification](https://www.dartlang.org/docs/spec/).
+
+Many tools in the Dart ecosystem use the
+[analyzer package](https://pub.dartlang.org/packages/analyzer) to
```

Comment:
You don't outright say that the analyzer package and the static analyzer are the same thing. We need to tie these together a bit more clearly.

Is this page about dartanalyzer, the analyzer package, the analysis server, ... ? (Actually, I don't see the analysis server mentioned anywhere, but shouldn't it be? It **is** mentioned in the /tools/ link to this page.)